n3fit/runcards/examples/nnpdf40-like_pos_cut.yaml

```diff
@@ -0,0 +1,189 @@
+#
```

Comment:
Please use a different name for this runcard. We should have only one `nnpdf40-like` to avoid generating confusion again.

Also, since this is just an example you can do with a subset of data.

Reply:
Yes but at the same time the `Basic_runcard.yml` does not have any MS-bar POS sets.
So I would rather have something called:
`Basic_runcard_ms_bar_pos_cuts.yaml`, where the runcard is the same as the current nnpdf40-like_pos_cut.yaml apart from reduced datasets.
What do you think?

Reply:
You can add a MS-bar POS set to the `Basic_runcard.yml`